sdk/internal/perf/testdata/perf/no_op_perf.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import "context"
+
+type noOpPerfTest struct{}
```

Comment:
I compared the no-op test results across go and .NET.  Go is about 15% slower, but it should be plenty fast for our scenarios:

![image](https://user-images.githubusercontent.com/9459391/150619322-42f4958d-9c8e-4444-8435-9fcbe28e0587.png)

Reply:
I have updated numbers for this, they're quite a bit higher now so I am verifying they are correct.